src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+@import "../utils/variables";
+
+.calendar {
+  padding: 0 10px;
+  max-width: ($day-size + $gap-value) * 7;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-value;
+
+  &__day {
+    height: $day-size;
+    width: $day-size;
+
+    display: flex;
+    box-sizing: border-box;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
```

Comment:
It`s all right. But maybe these parameters can be written on top of the main.scsss